wcivf/apps/elections/models.py

```diff
@@ -494,7 +494,9 @@ def party_ballot_count(self):
                 total_parties = ind_candidates + num_other_parties
                 return f"{total_parties} ballot options"
             else:
-                return f"{people.count()} candidate{pluralize(people.count())}"
+                winner_count = self.winner_count
+                winner_count_pluralized = pluralize(self.winner_count)
+                return f"{winner_count} ballot option{winner_count_pluralized}"
```

Comment:
I thought in this case it should be `candidate` rather than `ballot option`, as it is only list elections that you have ballot options? Might be worth checking with @symroe

Reply:
Checked w @pmk01, all good.